ma_statement.c

```diff
@@ -1312,12 +1309,13 @@ SQLRETURN MADB_StmtExecute(MADB_Stmt *Stmt)
 SQLRETURN MADB_StmtBindCol(MADB_Stmt *Stmt, SQLUSMALLINT ColumnNumber, SQLSMALLINT TargetType,
     SQLPOINTER TargetValuePtr, SQLLEN BufferLength, SQLLEN *StrLen_or_Ind)
 {
-  MADB_Desc *Ard= Stmt->Ard;
+  MADB_Desc *Ard;
   MADB_DescRecord *Record;
 
   if (!Stmt)
```

Comment:
This test is redundant. I guess it is better to remove it, and leave initialization as is